src/main.js

```diff
@@ -223,12 +224,16 @@ Formsy.Form = React.createClass({
   // validate the input and set its state. Then check the
   // state of the form itself
   validate: function (component) {
-
     // Trigger onChange
     if (this.state.canChange) {
       this.props.onChange(this.getCurrentValues(), this.isChanged());
     }
 
+    // SGG Hack: Formsy reeeeeallly really wants to validate.
```

Comment:
This is the first hack / fix. This fixes mounting / onChange to be moderately more performant.